config/installation_config.yml

```diff
@@ -235,6 +235,14 @@
   description: Used to notify Chatwoot about account abuses, potential threads (Should be a Discord Webhook URL)
 # ------- End of Chatwoot Internal Config for Self Hosted ----#
 
+# ------- Compliance Related Config ----#
+- name: CHATWOOT_INSTANCE_ADMIN_EMAIL
```

Comment:
add this config to the super admin internal config options (ref: Enterprise::SuperAdmin::AppConfigsController )